--- C:/Users/Jenso/Documents/codeQuake/WBB Pakete/de.codequake.page.start/de.codequake.page.start.index/de.codequake.page.start.header.patch31/header.tpl	Fri Aug 06 19:55:07 2010
+++ C:/Users/Jenso/Documents/codeQuake/WBB Pakete/de.codequake.page.start/de.codequake.page.start.index/de.codequake.page.start.header.patch31/header_patch.tpl	Fri Aug 06 19:55:51 2010
@@ -200,13 +200,13 @@
 		</div>
 		<div id="logo">
 			<div class="logoInner">
-				<h1 class="pageTitle"><a href="index.php?page=Index{@SID_ARG_2ND}">{lang}{PAGE_TITLE}{/lang}</a></h1>
+				<h1 class="pageTitle"><a href="index.php?page=Start{@SID_ARG_2ND}">{lang}{PAGE_TITLE}{/lang}</a></h1>
 				{if $this->getStyle()->getVariable('page.logo.image')}
-					<a href="index.php?page=Index{@SID_ARG_2ND}" class="pageLogo">
+					<a href="index.php?page=Start{@SID_ARG_2ND}" class="pageLogo">
 						<img src="{$this->getStyle()->getVariable('page.logo.image')}" title="{lang}{PAGE_TITLE}{/lang}" alt="" />
 					</a>
 				{elseif $this->getStyle()->getVariable('page.logo.image.application.use') == 1}
-					<a href="index.php?page=Index{@SID_ARG_2ND}" class="pageLogo">
+					<a href="index.php?page=Start{@SID_ARG_2ND}" class="pageLogo">
 						<img src="{@RELATIVE_WBB_DIR}images/wbb3-header-logo.png" title="{lang}{PAGE_TITLE}{/lang}" alt="" />
 					</a>
 				{/if}
